Anthropic.SDK.Tests/Anthropic.SDK.Tests.csproj

```diff
@@ -35,7 +35,7 @@
   <ItemGroup>
     <PackageReference Include="Google.Cloud.AIPlatform.V1" Version="3.24.0" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
-    <PackageReference Include="Microsoft.Extensions.AI" Version="9.3.0-preview.1.25161.3" />
+    <PackageReference Include="Microsoft.Extensions.AI" Version="9.4.0-preview.1.25207.5" />
```

Comment:
This PR should be more than just updating the version number, e.g. it should be setting MessageId, it should be using TextReasoningContent instead of the custom types, etc.

Reply:
> it should be setting MessageId, it should be using TextReasoningContent instead of the custom types

Can you please point me where and which custom types should be updated?  I'm not familiar with this project.  All I did (for this and all other projects) was bump the version and build, then fix source breaking changes, if any.